### PR TITLE
Add Lua as a language

### PR DESCRIPTION
**Pull request by soccermitchy**

### CODE REVIEW

File: lib/converter.js [hunk 1]
```diff
@@ -36,6 +36,16 @@ function jsonToLang(json, lang, level) {
       hashRowEnd: ",\n",
       indent: "  ",
       lineEnd: ""
+    },
+    lua: {
+      fileStart: "",
+      variable: "data = ",
+      hashStart: "{",
+      hashEnd: "}",
+      hashRow: "%s = %s"
```
Comment (vlucas) [inline lib/converter.js]: Lua doesn't need quotes for the key? What about if the key has spaces?

**Comment by soccermitchy** [comment]
Ah, didn't think about that. Is there a way I can update the PR?
Edit: just saw how

**Comment by vlucas** [comment]
Added this in a commit because it needed to be added to a variable in `app.js` too. Thanks!

**Comment by vlucas** [comment]
Commit: 169e839de8d98895329a8f2789d730d5f3e53f4f

**Pull request closed without merging**